src/dirac_cwl_proto/commands/download_config.py

```diff
@@ -0,0 +1,18 @@
+import os
+
+from dirac_cwl_proto.commands import CommandBase
+
+
+class DownloadConfig(CommandBase):
```

Comment:
Any reason that this can't just be a function ? I don't see much benefit of using abstract classes

@aldbr what do you think?